site/locals.tf

```diff
@@ -0,0 +1,27 @@
+locals {
+  service = {
+    live = {
+      suffix = ""
+      url    = "diagram.nationalarchives.gov.uk"
+    }
+    dev = {
+      suffix = "-dev"
+      url    = "dev-diagram.nationalarchives.gov.uk"
+    }
+  }
+  project_ns                     = "nata-dia3-${terraform.workspace}"
+  lambda_iam_name                = "lambda-iam"
+  lambda_function_name           = "diagram-backend"
+  lambda_timeout                 = 60
+  lambda_memsize                 = 5120
```

Comment:
Isn't this a bit too large?

Reply:
https://github.com/nationalarchives/DiAGRAM-terraform/pull/8#discussion_r2091541250